docs/CONTRACT.md

```diff
@@ -58,6 +58,11 @@ All in all there are 3 entities in the system:
 * A migration manager that, in an administrative capacity, handles migration destination contracts.
 * An emergency manager that, in an administrative capacity, can freeze the staking contract and override the cooldown period in the event of a serious bug in the contract.
 
+&nbsp;
+### Stake Change Notifications
+The staking contract implements an interface to notify another contract of a stake change. The stake change notification is designed to notify Orbs election contract, updating the staking and delegation map. The election contract is trusted not to revert a user staking transaction. Setting the notifier contract is done by the migration manager. 
```

Comment:
Maybe instead of "trusted", it'd be better to clearly state it as an accepted risk and explain that it'd be publicly audited and developed, as well as that in the worst case - the migration manager would be able to upgrade it.